### PR TITLE
Feat: Reminders in timeline

### PR DESCRIPTION
**Pull request by acalinica**
What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

## Summary by CodeRabbit

- **New Features**
    - Introduced the Merriweather font across the app for enhanced readability.
    - Added a new reminder management functionality within the FutureZone component, including creating, viewing, and dismissing reminders.
    - Implemented a due date picker for reminders, allowing users to select specific due dates and times.
    - Enhanced the email action workflow by simplifying the component structure and direct rendering of the ComposeEmailContainer.
- **Refactor**
    - Simplified the logic and structure of various Timeline actions (Email, Log Entry, Reminder) for improved maintainability and user experience.
    - Streamlined reminder management with GraphQL integration for queries and mutations.
- **Style**
    - Updated app layout to include new font styles and improved UI elements related to reminders.
- **Documentation**
    - Expanded type definitions to support new features like reminders and bank account management.

### CODE REVIEW

File: packages/apps/spaces/app/organization/[id]/src/components/Timeline/FutureZone/FutureZone.tsx [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+import { Reminders } from './reminders';
 
 export const FutureZone = () => {
-  return <div>{/* <Reminders /> */}</div>;
+  const isRemindersEnabled = useFeatureIsOn('reminders');
+
+  if (!isRemindersEnabled) return null;
+
+  return (
+    <>
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organization/[id]/src/components/Timeline/FutureZone/FutureZone.tsx]: Fragment not needed

File: packages/apps/spaces/ui/form/Textarea/FormAutoresizeTextarea.tsx [hunk 2]
```diff
@@ -21,6 +21,8 @@ export const FormAutoresizeTextarea = forwardRef<
 >(({ isLabelVisible, label, formId, ...props }, ref) => {
   const { getInputProps } = useField(props.name, formId);
 
+  // if (formId.startsWith('reminder')) console.log('VALUE', value);
```
Comment (Kasia9090) [inline packages/apps/spaces/ui/form/Textarea/FormAutoresizeTextarea.tsx]: 💅 comment can be removed

File: packages/apps/spaces/app/organization/[id]/src/components/Timeline/FutureZone/reminders/Reminders.tsx [hunk 3]
```diff
@@ -1,61 +1,132 @@
...
+            onChange={onChange}
+            onDismiss={onDismiss}
+            data={mapReminderToForm(r)}
+          />
+        ))}
     </VStack>
   );
 };
+
+function mapReminderToForm(
```
Comment (Kasia9090) [inline packages/apps/spaces/app/organization/[id]/src/components/Timeline/FutureZone/reminders/Reminders.tsx:119]: could that be DTO?

Comment (acalinica) [reply packages/apps/spaces/app/organization/[id]/src/components/Timeline/FutureZone/reminders/Reminders.tsx:119]: It coulsd, but I feel that it's a bit overkill.

**Review by Kasia9090 — APPROVED** [review]

**Pull request merged**